app/javascript/components/ResultsPanel.js

```diff
@@ -0,0 +1,119 @@
+    url: 'singlecell.broadinstitute.org/single_cell/api/v1/site/studies'
+  },
+  {type:'study',
+    name: 'Single nucleus RNA-seq',
+    cell_count: 5426,
+    acession: 'SCP3',
+    description: 'orem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
+  },
+  {
+    type:'study',
+    name: 'Single nucleus RNA-seq',
+    cell_count: 5426,
+    acession: 'SCP4',
+    description: 'orem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
+  },
+  {type:'study',
+    name: 'Single nucleus RNA-seq',
+    cell_count: 5426,
+    acession: 'SCP5',
+    description: 'orem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
+  },
+];
+
+class ResultsPanel extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      results: undefined,
+      facets : []
+    };
+    this.handleStudyLabel = this.handleStudyLabel.bind(this);
+    };
+
```

Comment:
maybe rename to handleStudyLabelClick?

Reply:
Also, I think this method belongs in the Study component, rather than here.  (assuming the study click function does a expansion of the study block similar  to the functionality  in the current site)

Reply:
This will actually just be an <a>. This has been reflected in the new PR.